Simulator/wdc-simulator.js

```diff
@@ -933,21 +933,13 @@
     },
     
     getDataElements(tableData, schema) {
-      var dataTableRowKey = 0;
-      var dataTableColKey = 0;
+      var dataTableRowKey = 1;  
       var dataElements = [];
       if (tableData) { // We may not fetched any data yet
         dataElements = tableData.slice(0, TablePreview.MAX_ROWS).map(function(row) {
-          dataTableColKey = 0;
           return DOM.tr({ key: dataTableRowKey++ },
             schema.map(function(header) {
-              if (!row[header]) {
```

Comment:
Thanks for the PR @Jaxolotl!

We need to keep extra logic that is currently there because we need to support two use cases:
1. WDC sends an array of objects back through table.appendRows
2. WDC sends an array of arrays back through table.appendRows

For (1), we can map things correctly, but for (2), we just through the data into the row as is.

The issue here is that I have a bug in this code path that prevents the code path for (1) to be executed.  All that needs to happen is this line should instead be `if (_.isUndefined(row[header])) {`.

And the other change (with adding alias) looks great!